[2023-01-23T14:29:05.190-0300] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: get_antigens.catnap manual__2023-01-23T17:29:02.353260+00:00 [queued]>
[2023-01-23T14:29:05.205-0300] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: get_antigens.catnap manual__2023-01-23T17:29:02.353260+00:00 [queued]>
[2023-01-23T14:29:05.206-0300] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-01-23T14:29:05.206-0300] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2023-01-23T14:29:05.206-0300] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-01-23T14:29:05.230-0300] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): catnap> on 2023-01-23 17:29:02.353260+00:00
[2023-01-23T14:29:05.235-0300] {standard_task_runner.py:55} INFO - Started process 5388 to run task
[2023-01-23T14:29:05.237-0300] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'get_antigens', 'catnap', 'manual__2023-01-23T17:29:02.353260+00:00', '--job-id', '340', '--raw', '--subdir', 'DAGS_FOLDER/get_antigens.py', '--cfg-path', '/tmp/tmpbcf8yclf']
[2023-01-23T14:29:05.238-0300] {standard_task_runner.py:83} INFO - Job 340: Subtask catnap
[2023-01-23T14:29:05.325-0300] {task_command.py:389} INFO - Running <TaskInstance: get_antigens.catnap manual__2023-01-23T17:29:02.353260+00:00 [running]> on host kallo-Ubuntu
[2023-01-23T14:29:05.415-0300] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Kallox
AIRFLOW_CTX_DAG_ID=get_antigens
AIRFLOW_CTX_TASK_ID=catnap
AIRFLOW_CTX_EXECUTION_DATE=2023-01-23T17:29:02.353260+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-23T17:29:02.353260+00:00
[2023-01-23T14:29:07.992-0300] {python.py:177} INFO - Done. Returned value was: None
[2023-01-23T14:29:08.006-0300] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=get_antigens, task_id=catnap, execution_date=20230123T172902, start_date=20230123T172905, end_date=20230123T172908
[2023-01-23T14:29:08.056-0300] {local_task_job.py:159} INFO - Task exited with return code 0
[2023-01-23T14:29:08.091-0300] {taskinstance.py:2582} INFO - 1 downstream tasks scheduled from follow-on schedule check
